### PR TITLE
[risk=no] Prod Cdr update 

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
Prod Cdr Update

### CODE REVIEW

File: public-ui/dispatch.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+dispatch:
+  - url: "aou-db-staging.appspot.com/v1/*"
+    service: api
+  - url: "*/favicon.ico"
+    service: default
+  - url: "aou-db-staging.appspot.com/"
+    service: default
```
Comment (mhigbyflowers) [inline public-ui/dispatch.yaml]: a new github feature let me know that there needs to be a new line at the end of the file.

File: public-ui/iap_settings.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+accessSettings:
+  corsSettings:
+    allowHttpOptions: false
```
Comment (mhigbyflowers) [inline public-ui/iap_settings.yaml]: here too

**Review by mhigbyflowers — APPROVED** [review]
small nits. 👍

**Comment by SrushtiGangireddy** [comment]
Thanks

**Pull request merged**